pytroll_collectors/segments.py

```diff
@@ -406,7 +408,18 @@ def is_relevant(self, message):
         if mask in slot_pattern['received_files']:
             logger.debug("File already received")
             should_be_added = False
-        if mask not in slot_pattern['all_files']:
+        if mask in slot_pattern['all_files']:
+            pass
```

Comment:
It maybe me, but I get a bit confused using a pass statement like this, wouldn't it be more clear to just do a 
`return mask, should_be_added` here directly instead!?

Reply:
@pnuu or @mraspaud may have different opinions?

Reply:
I don't have a strong opinion. The other branches do checks and modifies `should_be_added` accordingly and don't do a return.

Reply:
If we can return immediately it's best to do it, yes.